frontend/packages/process-editor/src/components/Canvas/Canvas.tsx

```diff
@@ -41,17 +41,15 @@ export const Canvas = ({ onSave, appLibVersion }: CanvasProps): JSX.Element => {
   const isEditAllowed: boolean = supportsProcessEditor(appLibVersion);
 
   return (
-    <>
+    <div className={classes.container}>
       {(isEditAllowed || shouldDisplayFeature('shouldOverrideAppLibCheck')) && (
```

Comment:
One of my previous comments suggested moving the `appLibVersion` check to the `ContextProvider`, if we do so we can add the `shouldDisplayFeature('shouldOverrideAppLibCheck')` into the `ContextProvider` as well so we can override the `appLibVersion` check while developing process-editor everywhere the `isEditAllowed` is used from the `ContextProvider`. 😊